lib/rules/require-spaces-in-anonymous-function-expression.js

```diff
@@ -42,6 +43,10 @@ module.exports = function() {};
 
 module.exports.prototype = {
     configure: function(options) {
+        this.exceptions = {
```

Comment:
We pretty strict about naming conventions of the private properties, could name it `._exceptions`?